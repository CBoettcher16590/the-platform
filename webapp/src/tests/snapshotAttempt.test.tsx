import * as React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Login from '../pages/forms/signin';

test("should render correctly", () => {
  const wrapper = shallow(<Login />).toJSON();
  expect(toJson(wrapper)).toMatchSnapshot();
});


import { CountDown } from "./CountDown";

test("should render correctly", () => {
  const wrapper = shallow(<CountDown startTime={30} />).toJSON();
  expect(toJson(wrapper)).toMatchSnapshot();
});








// import * as React from "react";
// import * as renderer from "react-test-renderer";
// import Login from '../pages/forms/signin';


// describe('<CheckoutFormYear as a snapshot??????', () => {

//     test("should render correctly", () => {
//     const tree = renderer.create(<Login />).toJSON();
//     expect(tree).toMatchSnapshot();
//     });

// })
