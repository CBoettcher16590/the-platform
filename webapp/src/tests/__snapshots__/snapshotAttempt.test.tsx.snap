// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snap shot test method two. 1`] = `
<div
  className="loginSignupBackground"
>
  <div
    id="loginForm"
  >
    <h1>
      Log In
    </h1>
    <form
      className=""
    >
      <div
        className="form-group"
      >
        <label
          className="form-label"
        >
          Email Address
        </label>
        <input
          className="form-control form-control"
          id="emailInput"
          onChange={[Function]}
          placeholder="Enter email"
          type="email"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
        >
          Password
        </label>
        <input
          className="form-control form-control"
          onChange={[Function]}
          placeholder="Enter password"
          type="password"
          value=""
        />
      </div>
      <button
        className="btn btn-primarys btn-block btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="submit"
      >
        Sign In
      </button>
      <a
        href="#"
      >
        Forgot Your Password?
      </a>
    </form>
    <hr />
    <div
      className="createAccount"
    >
      <h4>
        Dont have an Account Yet?
      </h4>
      <a
        href="/signup"
      >
        Create an Account
      </a>
    </div>
  </div>
</div>
`;

exports[`Snap shot test one. 1`] = `
<div
  className="loginSignupBackground"
>
  <div
    id="loginForm"
  >
    <h1>
      Log In
    </h1>
    <form
      className=""
    >
      <div
        className="form-group"
      >
        <label
          className="form-label"
        >
          Email Address
        </label>
        <input
          className="form-control form-control"
          id="emailInput"
          onChange={[Function]}
          placeholder="Enter email"
          type="email"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
        >
          Password
        </label>
        <input
          className="form-control form-control"
          onChange={[Function]}
          placeholder="Enter password"
          type="password"
          value=""
        />
      </div>
      <button
        className="btn btn-primarys btn-block btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="submit"
      >
        Sign In
      </button>
      <a
        href="#"
      >
        Forgot Your Password?
      </a>
    </form>
    <hr />
    <div
      className="createAccount"
    >
      <h4>
        Dont have an Account Yet?
      </h4>
      <a
        href="/signup"
      >
        Create an Account
      </a>
    </div>
  </div>
</div>
`;
